src/containers/offer-page/chat-dialog-window/ChatDialogWindow.tsx

```diff
@@ -89,14 +101,40 @@ const ChatDialogWindow: FC<ChatDialogWindow> = ({ chatInfo }) => {
     messagesListWithDate
   )
 
-  const questionsList = questions.map((el) => (
-    <Typography key={el.question} sx={styles.question}>
-      {t(el.question)}
-    </Typography>
-  ))
+  const questionsList = creatingChat ? (
+    <Box sx={styles.chatCreateBox}>
+      <Loader size={20} />
+      <Typography sx={styles.question}>{t('chatPage.creating')}</Typography>
+    </Box>
+  ) : (
+    questions.map((el) => (
+      <Typography key={el.question} sx={styles.question}>
+        {t(el.question)}
+      </Typography>
+    ))
+  )
 
   const closeChatWindow = () => setChatInfo(null)
 
+  const openChatInNewTab = (chatId: unknown) => {
+    localStorage.setItem('currentChatId', chatId as string)
+    window.open(authRoutes.chat.path, '_blank', 'noopener noreferrer')
+  }
+
+  const handleCreateNewChat = async () => {
+    setCreatingChat(true)
+    const newChat = await createChat()
+    openChatInNewTab(newChat.data._id)
```

Comment:
what if there will be an error during the request? Maybe make sense to use useAxios hook for handling an error and take loading from it, check it

Reply:
> what if there will be an error during the request? Maybe make sense to use useAxios hook for handling an error and take loading from it, check it

You are right. Fixed!

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/20dc456e-3ad6-48a9-a2e6-41f767df6ea9